packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -106,7 +106,7 @@ module.exports = class SingleSpaReactGenerator extends Generator {
       templateOptions
     );
     this.fs.copyTpl(
-      this.templatePath("../../common-templates/.gitignore"), // this is relative to /templates
+      this.templatePath("../../common-templates/gitignore"), // this is relative to /templates
```

Comment:
This is strange. 

NPM reads all .gitignore files and turns them into a .npmignore so in the packaged code there is no .gitignore but if we change this to `gitignore` without the `.` then NPM won't mess with it.